src/components/table/footer/style.css

```diff
@@ -0,0 +1,12 @@
+:host {
+	display: table-footer-group;
+	cursor: default;
+	position: sticky;
+	bottom: 0;
+	background: white ;
```

Comment:
the background should be a css variable. Either one of the `--smoothly-color-whatever` ones or one of the ones defined in the common.css file

Reply:
Will fix!